float32 ttc_min
float32 ttc_min_angle